docs/aerodromes/Jandakot.md

```diff
@@ -0,0 +1,135 @@
+
+All aircraft will arrive on runway 06L/24R or 12/30.
+
+!!! note
+    Circuit joining instructions given without an assigned altitude imply clearance to conduct the visual approach. There is no need to clear these aircraft for a visual approach.
+
+### VFR Outbound Procedures
+VFR aircraft should report ready to **JT ADC** with their departure intentions.  A takeoff clearance constitutes a clearance to depart the zone by extending the pilot's requested leg of the circuit.  Aircraft departing the zone into class G airspace will transfer to area frequency upon leaving the zone, **no explicit frequency transfer is given to these aircraft**.
+
+Aircraft departing a leg of the circuit will climb to and maintain the following levels until clear of the zone:  
+All runways: `A010`  
+
+VFR aircraft will depart via set outbound departure routes. Aircraft will track extended circuit legs to the departure point. These departure points include: `YGB`, `FREM` via `Fiona Stanley Hospital`, and `SHOP`.
+
+Departures via `FREM` and `YGB` will depart on runway 06L/24R. Runway 06R/24L is used for circuit traffic and departures via `SHOP`.
+
+### Circuits
+The circuit direction changes depending on tower opening hours and runway being used.
+
+| Runway | Tower Open   | Tower Closed |
+| -------| ----- | ----- |
+| 06L    | Left  | Right |
+| 06R    | Right | N/A   |
+| 24L    | Left  | N/A   |
+| 24R    | Right | Left  |
+| 12     | Left  | Left  |
+| 30     | Left  | Left  |
+
+Circuits to be flown at `A010`
+
+## IFR Inbound Procedures
+Aircraft arriving from the Perth Class C in VMC will track via `CNB`. These aircraft will contract **JT ADC** approaching `CNB` and will be issued circuit joining instructions.  
+Aircraft in IMC will track inbound via an instrument approach.  
+**PH TCU** will coordinate all IFR arrivals in accordance with coordination procedures.
+
+## Helicopter Operations
+### General
+Unless otherwise depicted in the `ERSA FAC YPJT`, all helicopters must comply with fixed wing procedures.
+
+### Circuits
+Circuits are conducted within the lateral confines of the fixed-wing circuit at `A008`, in the same direction as the current runway config.  The termination point of the circuit is the Main Pad.
+
+
+## ATIS
+### Runway Nomination
+The ATIS must indicate the current runway config and nominate what each parallel runway is being used for.  The northern runway (06L/24R) is primarily used for VFR arrivals and departures, the southern runway (06R/24L) for circuit training and departures via `SHOP`, and the cross runway (12/30) for arrivals/departures when weather is unsuitable for parallel rwunay operations.  
+
+This should be reflected on the ATIS as below:  
+`RWY 06L FOR ARRIVALS AND DEPARTURES; RWY 06R FOR CIRCUITS AND DEPARTURES VIA ARMADALE SHOPS`  
```

Comment:
These should be other way around.

Reply:
Sorry which part? You might have your lefts and rights mixed up

Reply:
should be `RWY 06R FOR CCTS AND DEPS VIA SHOP, RWY 06L FOR ARRIVALS AND ALL OTHER DEPS.` They always mention the runway with deps via Armadale first. Just had a listen to the ATIS then while they are running 06 ops to confirm.